server/main.go

```diff
@@ -13,6 +13,15 @@ import (
 var ggcDir string
 var port string
 
+func ggcDirDefault() string {
+	dir := "ggc"
+	b, err := exec.Command("git", "config", "ghq.root").CombinedOutput()
+	if err == nil {
+		dir = string(b)
```

Comment:
Is not unnecessary new line character appended?
I think it will make some defect.